.clang-tidy

```diff
@@ -0,0 +1,8 @@
+---
+# TODO: Enable as many useful checks as possible.
+# E.g. set to '*', see which of those aren't useful, and disable just those.
+Checks: '*,-*,llvm-include-order'
```

Comment:
clang-format already sorts includes.  Probably not a good idea to have this check in both, because then if they ever happen to disagree, they will fight and we will be the losers.  :)